lib/airport.rb

```diff
@@ -1,2 +1,51 @@
 class Airport
-end
+
+  MAXIMUM_CAPACITY = 10
+
+  attr_accessor :planes, :capacity,
+
+  def initialize
+    @capacity = (:capacity)
+    @planes = []
+  end
+
+
+  def capacity
+    @capacity = MAXIMUM_CAPACITY
+  end
+
+  def planes
+    @planes = 4
+  end
+
+  def land(planes)
+    receive "landing prohibited in a storm!" if stormy?
+    receive "Airport is full" if full?
+    receive "plane has landed" if has_landed
+    receive "ready to take_off?" if clear_skies
+  end
+
+  def take_off(planes)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `planes`. If it's necessary, use `_` or `_planes` as an argument name to indicate that it won't be used. You can also write as `take_off(*)` if you want the method to accept any arguments but don't care about them.